modules/snooker/src/test/ts/module/ephox/snooker/test/Bridge.ts

```diff
@@ -51,7 +51,24 @@ const generators: Generators = {
   colgroup: () => SugarElement.fromTag('colgroup')
 };
 
+const createCell = (): SugarElement<HTMLTableDataCellElement> => {
+  const tag = SugarElement.fromTag('td');
+  Insert.append(tag, SugarElement.fromText('?'));
+  return tag;
+};
+
+// Added this for testing pasting and merging as Fitmet.tailor expects cell to not rely on previous cell
```

Comment:
```suggestion
// Added this for testing pasting and merging as Fitment.tailor expects cell to not rely on previous cell
```

Reply:
Have changed the comment so the reasoning for it hopefully makes more sense